### DISCUSSION TITLE
Working with Palma Hydrojet (S200102)

### DISCUSSION THREAD

**Discussion by urbanframe** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#458] [Show and tell]
@visualapproach - A huge thank you and well done on this project. 

I can confirm I have successfully set up the Palma Hydrojet Pro (S200102 square pump). 

I'm using the blue TXS0108E LLC, 6 wires, wired as per page 8 of your guide

![image](https://user-images.githubusercontent.com/8011551/221920561-cc4e2df0-4996-4d80-b86a-5bd1b8abb68d.jpeg)

I've selected MALDIVES2021 //hydrojets in Model.h

Thanks again!

**Comment by visualapproach** [comment]
That's a lovely kind of blue there 🤣 (Yeah, I know it's from the manual, but I couldn't help myself)
Sorry, I appreciate your words and for reporting success! Good job!

**Reply by urbanframe** [reply]
I didn't think to take a photo before I replaced the lid on the pump!! 😂. I also forgot to enable ota updates so will be opening sooner than I planned. 

Thanks again for revolutionising our spa time!

**Reply by visualapproach** [reply]
You are welcome! The OTA is on by default so unless you turned it off somehow, it should work.
